packages/vaadin-lumo-styles/utilities/background.js

```diff
@@ -15,106 +15,106 @@ export const background = css`
     background-color: transparent;
   }
 
-  .bg-tint-5 {
-    background-color: var(--lumo-tint-5pct);
+  .bg-tint {
```

Comment:
Not sure why this sorting is really needed but I would prefer to extract it from this PR, so as this would only be about adding new utility classes (easier to review and read it in the future).

Reply:
OCD. I wanted the LumoUtility classes and the actual CSS to line up. Easier to manage that way.